### ISSUE TITLE
Javascript driver + appium giving error

### ISSUE THREAD

**Issue by pritamkarmakar** [appium/appium#4332]
Getting this error. I have Android platform 5.0.1 and appium 1.3.4 installed in my machine

 Error: [init({"browserName":"","appium-version":"1.3.4","platformName":"Android","platformVersion":"5.0.1","deviceName":"Android Emulator","app":"http://appium.github.io/appium/assets/ApiDemos-debug.apk"})] The environment you requested was unavailable.

**Comment by sebv** [comment]
Works fine for me with this exact capabilities. Could you post full server log in a [gist](https://gist.github.com/).

**Comment by pritamkarmakar** [comment]
Sorry I forgot to update this thread, this worked for me as well. When I checked the log I saw that my emulator is not running, thats why it was giving me this error. 

One thing I noticed it is not documented in the samples readme.md file that user need to update the 'helpers > caps.js' file with the correct version info
```bash
'appium-version': '1.3.4',
  platformName: 'Android',
  platformVersion: '5.0.1',
````

Thanks for looking into it, really appreciate it.
